### ISSUE TITLE
Command line interface

### ISSUE THREAD

**Issue by c-martinez** [NLeSC/ShiCo#29]
It would be nice to have a command line tool which generated vocabularies.

**Comment by MacTommyK** [comment]
Isn't this what my original code did/does?

Tom

**Comment by c-martinez** [comment]
Yes and no -- I thought it would be nice to have a command line interface, but which connects to the running server. So you don't need a big machine locally to do the heavy lifting.

**Comment by MacTommyK** [comment]
Ah, that's what you mean. Yes. Excellent idea!!

Tom
